src/components/Tabs/Tabs.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+import { Button } from 'semantic-ui-react';
+
+class Tabs extends Component {
+
+  render() {
+    return (
+      <>
+        {this.props.tabs.map(el => (
+          <button
+            onClick={() => this.props.onTabSelected(el.content)}
+            class="ui positive button">
```

Comment:
```suggestion
            className="ui positive button">
```

https://reactjs.org/docs/dom-elements.html#classname